machine_learning/mfcc.py

```diff
@@ -158,6 +166,14 @@ def audio_frames(
 
     Returns:
         np.ndarray: An array of overlapping frames.
+    
+    Examples:
+        >>> import numpy as np
+        >>> audio = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]*1000)
+        >>> sample_rate = 8000
+        >>> frames = audio_frames(audio, sample_rate, hop_length=10, ftt_size=512)
+        >>> frames.shape
+        (126, 512)
```

Comment:
Decrease the indent to give yourself a bit more room to write tests.
```suggestion
    >>> import numpy as np
    >>> audio = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]*1000)
    >>> sample_rate = 8000
    >>> frames = audio_frames(audio, sample_rate, hop_length=10, ftt_size=512)
    >>> frames.shape
    (126, 512)
```

Reply:
Done.